yahoo.com

```diff
@@ -23,30 +22,36 @@
       <port>993</port>
       <socketType>SSL</socketType>
       <username>%EMAILADDRESS%</username>
+      <authentication>OAuth2</authentication>
       <authentication>password-cleartext</authentication>
     </incomingServer>
     <incomingServer type="pop3">
       <hostname>pop.mail.yahoo.com</hostname>
       <port>995</port>
       <socketType>SSL</socketType>
       <username>%EMAILADDRESS%</username>
+      <authentication>OAuth2</authentication>
       <authentication>password-cleartext</authentication>
     </incomingServer>
     <outgoingServer type="smtp">
       <hostname>smtp.mail.yahoo.com</hostname>
       <port>465</port>
       <socketType>SSL</socketType>
       <username>%EMAILADDRESS%</username>
+      <authentication>OAuth2</authentication>
       <authentication>password-cleartext</authentication>
     </outgoingServer>
   </emailProvider>
   <webMail>
-    <loginPage url="https://mail.yahoo.com" />
+    <loginPage url="https://mail.yahoo.com"/>
     <loginPageInfo url="https://mail.yahoo.com/">
       <username>%EMAILADDRESS%</username>
-      <usernameField id="login-username" />
-      <passwordField id="login-passwd" />
-      <loginButton id="login-signin" />
+      <usernameField id="login-username"/>
+      <passwordField id="login-passwd"/>
+      <loginButton id="login-signin"/>
     </loginPageInfo>
   </webMail>
+  <documentation url="https://help.yahoo.com/kb/new-mail-for-desktop">
```

Comment:
This is just the general help site. Please link to the specific pages for Thunderbird, e.g.
https://help.yahoo.com/kb/new-mail-for-desktop/imap-server-settings-yahoo-mail-sln4075.html?impressions=true
and
https://help.yahoo.com/kb/new-mail-for-desktop/pop-access-settings-instructions-yahoo-mail-sln4724.html?impressions=true

Reply:
(FWIW, pay attention to escape "?" in XML.)